src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -362,6 +446,12 @@ private void InsertEntry(NodeSearchElementViewModel entry, IEnumerable<string> c
             target.Entries.Add(entry);
         }
 
+        private static string AddToFullPath(string path, string addition)
```

Comment:
After debugging, I think a better name for this would be `MakeFullyQualifiedName`.